rust/src/types/amount.rs

```diff
@@ -157,6 +170,30 @@ impl From<usize> for u256 {
     }
 }
 
+impl From<Address> for u256 {
```

Comment:
I pressure that these conversions simplify quite a lot popping/pushing addresses in the stack. 
I wonder if the Address type should be a Cow, to avoid copying in read-only situations. You can add this to the list to investigate in the future.

Reply:
`Address` is almost always used as an argument for db functions so the conversion needs to happen anyway.